web/sites/default/config/search_api.index.full_index.yml

```diff
@@ -237,6 +250,14 @@ field_settings:
     dependencies:
       config:
         - field.storage.node.field_history
+  field_location:
```

Comment:
This field is under "Group" not "Content". In `/admin/config/search/search-api/index/full_index/fields`, I tried to added `Location » Content » Geolocation` for Content but can't expand beyond the `field_location -> Content` to reach the Geolocation field.